app/views/ExposureHistory/index.tsx

```diff
@@ -61,20 +61,16 @@ const ExposureHistoryScreen = (): JSX.Element => {
     selectedDatum && !DateTimeUtils.isInFuture(selectedDatum.date);
 
   return (
-    <SafeAreaView>
-      <ScrollView
-        style={styles.container}
-        contentInset={{ top: 0, bottom: 140 }}>
+    <SafeAreaView style={{ flex: 1 }}>
```

Comment:
Ah I get what you were explaining earlier now, I thought you wanted to put the flex on the scroll view directly.  This makes a lot of sense. 👍